### PR TITLE
feature(actors): add CreateTimer unit tests

### PR DESCRIPTION
**Pull request by 1046102779**
Add actors/timer UT

### CODE REVIEW

**Comment by 1046102779** [comment]
please @artursouza this PR

File: pkg/actors/actors_test.go [hunk 1]
```diff
@@ -262,6 +262,92 @@ func TestCreateReminder(t *testing.T) {
...
+	go testActorsRuntime.startTimer(ctx, stop,
+		&CreateTimerRequest{
+			Name:      "timer1",
+			ActorID:   actorID,
+			ActorType: actorType,
+			DueTime:   "5s",
+			Period:    "2s",
+			Data:      nil,
+		})
+	time.Sleep(8 * time.Second)
```
Comment (artursouza) [inline pkg/actors/actors_test.go:349]: Can this test that the timer call actually happened. Make it more frequent, so it does not need to sleep for 8 seconds. That is a lot of time.

**Review by artursouza — CHANGES_REQUESTED** [review]

**Comment by artursouza** [comment]
@1046102779  There are some conflicts now in actors_test.go.

File: pkg/actors/actors_test.go [hunk 2]
```diff
@@ -284,6 +284,92 @@ func TestCreateReminder(t *testing.T) {
...
+	go testActorsRuntime.startTimer(ctx, stop,
+		&CreateTimerRequest{
+			Name:      "timer1",
+			ActorID:   actorID,
+			ActorType: actorType,
+			DueTime:   "2s",
+			Period:    "1s",
+			Data:      nil,
+		})
+	time.Sleep(3 * time.Second)
```
Comment (artursouza) [inline pkg/actors/actors_test.go:371]: Please, modify the test so it does not depend on sleep and add validation.

/cc @pkedy

**Review by artursouza — CHANGES_REQUESTED** [review]
I am still concerned about using Sleep() in unit tests.

**Pull request closed without merging**